src/fontra/views/editor/scene-controller.js

```diff
@@ -121,6 +121,7 @@ export class SceneController {
 
     // Set up the mutual dependencies between location and selectedSourceIndex
     this.sceneSettingsController.addKeyListener("location", async (event) => {
+      this.setCanvasFontVariables();
```

Comment:
None of this belongs here, it should all be in panel-reference-font.js, like I wrote before.

Reply:
I moved all of them to reference fonts panel.